crt_portal/cts_forms/model_variables.py

```diff
@@ -1,58 +1,59 @@
+        _('Denied housing, a permit, or a loan'),
+        _('Harmful living conditions or lack accommodations for disability'),
+        _('Harassment by a landlord or another tenant'),
     ],
     'education': [
-        'Harassment based on race, sex, national origin, disability, or religion',
-        'Denied admission or segregated in an education program or activity',
-        'Denied services or accommodations for a disability or language barrier'
+        _('Harassment based on race, sex, national origin, disability, or religion'),
+        _('Denied admission or segregated in an education program or activity'),
+        _('Denied services or accommodations for a disability or language barrier'),
     ],
     'voting': [
-        'Blocked from registering to vote, entering a polling place to vote, or any other voting activity',
-        'Lack of polling place accommodations for disability',
-        'Ballot tampering'
+        _('Blocked from registering to vote, entering a polling place to vote, or any other voting activity'),
+        _('Lack of polling place accommodations for disability'),
+        _('Ballot tampering'),
     ],
     'police': [
-        'Police brutality or use of excessive force, including patterns of police misconduct',
-        'Searched and arrested under false pretenses, including racial or other discriminatory profiling',
-        'Denied rights, language access barriers, subjected to harmful living conditions or lack of accessible facilities'
+        _('Police brutality or use of excessive force, including patterns of police misconduct'),
+        _('Searched and arrested under false pretenses, including racial or other discriminatory profiling'),
+        _('Denied rights, language access barriers, subjected to harmful living conditions or lack of accessible facilities'),
     ],
     'commercial_or_public': [
-        'A location or website lacking disability accommodations',
-        'Denied service or entry because of a percieved personal characteristic like race, sex, or religion',
-        'Blocked from receiving reproductive health services',
+        _('A location or website lacking disability accommodations'),
+        _('Denied service or entry because of a perceived personal characteristic like race, sex, or religion'),
+        _('Blocked from receiving reproductive health services'),
     ],
     'something_else': []
 }
 
-# This will create the initial order, the form_order data can be directly adjusted after the initial load.
+# PROTECTED_CLASS_CHOICES means "PROTECTED_CLASS_FORM_CHOICES" and refers to the choices that will be displayed on the form front-end.
 # See protected maintenance docs: https://github.com/usdoj-crt/crt-portal/blob/develop/docs/maintenance_or_infrequent_tasks.md#change-protected-class-options
+# This tuple will create the initial order, the form_order data can be directly adjusted after the initial load.
 PROTECTED_CLASS_CHOICES = (
```

Comment:
Used this branch to try marking Protected Class strings for translation using `gettext_lazy`.

I found that this approach did not work — on startup, a translated version of the Protected Class choice was automatically added by Django to the database, which is not the behavior we want to see. See DATA ACTION log item in screenshot:

<img width="1435" alt="translation-db-startup" src="https://user-images.githubusercontent.com/3209501/69381412-56cc6280-0c7a-11ea-9d3c-daaeed6cfc4c.png">

We'll need to either take a closer look at the Protected Class model or use a library to translate the string for users while keeping only a single language option in the database; @LindsayYoung found `django-modeltranslation`, which looks promising: https://github.com/deschler/django-modeltranslation